pctiler/pctiler/colormaps/__init__.py

```diff
@@ -22,6 +23,7 @@
     **lulc_colormaps,
     **mtbs_colormaps,
     **chloris_colormaps,
+    "noaa-c-cap": noaa_c_map_colormap,
```

Comment:
@mmcfarland do you have a preference around maintaining the `**x_colormaps` approach for consistency?

Reply:
I would prefer that, especially if we may over time be adding noaa related cmaps